tools/ios/build_xcframework.py

```diff
@@ -179,6 +191,20 @@ def parse_args():
         required=True,
         help="Path to output directory.",
     )
+    # Same as how android works
+    # This one is used in the ci pipeline for accelerating the build process,
+    # We have 4 archs to be build. It's in sequence by default, but we can build them in parallel.
+    # The parallel build works as:
+    #   1. build the lib files for each arch in different ci jobs
+    #   2. download all the lib files in tasks
+    #   2. pack the xcframework package in ci task
+    parser.add_argument(
+        "--build_target",
+        default="xcframework",
+        choices=["lib", "xcframework", "pack_xcframework"],
+        help="Build target. 'xcframework' builds the whole package. 'lib' builds the libraries only. \
+            'pack_xcframework' only does pack xcframework from existing lib files.",
```

Comment:
nit: "target" has multiple meanings and it's not obvious to me what it means here.

just an idea for another name: `--mode` option with values `build_platform_arch_frameworks_only`, `pack_xcframework_only`, `build_xcframework` (default)

similar comment for build_aar.py